BookingSystem/Frontend/src/hotelSearch/containers/HotelSearch.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { HotelSearchActions } from "../actions";
+import HotelSearch from "../components/HotelSearch";
+
+const mapStateToProps = state => {
+  const hotelSearch = state.hotelSearch;
+  return {
+    hotels: hotelSearch.hotels,
+    isFetching: hotelSearch.isFetching,
+    error: hotelSearch.error
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  const bindedCreators = bindActionCreators(
+    {
+      getHotels: HotelSearchActions.fetchHotels
+    },
+    dispatch
+  );
+  return { ...bindedCreators };
+};
+
+export default connect(
```

Comment:
I don't quite understand why you need this container at least on this stage, it is useless